src/server/pps/server/defaults.go

```diff
@@ -322,3 +352,42 @@ func unmarshalJSON(s string, v any) error {
 	d.UseNumber()
 	return errors.Wrapf(d.Decode(v), "could not unmarshal %q as JSON", s)
 }
+
+func staticallyValidate(req *pps.CreatePipelineRequest) error {
+	// TODO(msteffen) eventually TFJob and Transform will be alternatives, but
+	// currently TFJob isn't supported
+	if req.TfJob != nil {
+		return errors.New("embedding TFJobs in pipelines is not supported yet")
+	}
+	if !(req.ReprocessSpec == "" || req.ReprocessSpec == client.ReprocessSpecUntilSuccess && req.ReprocessSpec != client.ReprocessSpecEveryJob) {
```

Comment:
shouldn't the last conditional be `==` instead of `!=`?

Reply:
Yeah, and a `||` too.

I fixed a test for the permissible values, as well.